platform-plugins.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: page
+title: Platform-specific APIs and Platform Plugins
```

Comment:
We don't have great answers to these questions yet; we need to write some more plugins first.